### DISCUSSION TITLE
Doc comment on ANN Benchmark (/benchmarks/ann)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1874] [Docs/blog comments]
# Doc comment on ANN Benchmark (/benchmarks/ann)

This vector database benchmark is designed to measure and illustrate Weaviate's Approximate Nearest Neighbor (ANN) performance for a range of real-life use cases.

https://weaviate.io/developers/weaviate/benchmarks/ann

**Comment by yy642ang** [comment]
What does limit 1, limit 10, and limit 100 mean ?

**Reply by databyjp** [reply]
Hi @yy642ang - 
I believe it refers to how many top N results are being fetched to compare to the ground truth, for use in the benchmarks. The number affects search performance, and determines the basis for recall measurements.
